symengine/lib/symengine_wrapper.pyx

```diff
@@ -3886,5 +4176,102 @@ def ccode(expr):
     cdef Basic expr_ = sympify(expr)
     return symengine.ccode(deref(expr_.thisptr)).decode("utf-8")
 
+
+def piecewise(*v):
+    cdef symengine.PiecewiseVec vec
+    cdef pair[RCP[symengine.const_Basic], RCP[symengine.const_Boolean]] p
+    cdef Basic e
+    cdef Boolean b
+    for expr, rel in v:
+        e = sympify(expr)
+        b = sympify(rel)
+        p.first = <RCP[symengine.const_Basic]>e.thisptr
+        p.second = <RCP[symengine.const_Boolean]>symengine.rcp_static_cast_Boolean(b.thisptr)
+        vec.push_back(p)
+    return c2py(symengine.piecewise(symengine.std_move_PiecewiseVec(vec)))
+
+
+def interval(start, end, left_open=False, right_open=False):
+    cdef Number start_ = sympify(start)
+    cdef Number end_ = sympify(end)
+    cdef cppbool left_open_ = left_open
+    cdef cppbool right_open_ = right_open
+    cdef RCP[const symengine.Number] n1 = symengine.rcp_static_cast_Number(start_.thisptr)
+    cdef RCP[const symengine.Number] n2 = symengine.rcp_static_cast_Number(end_.thisptr)
+    return c2py(symengine.interval(n1, n2, left_open_, right_open_))
+
+
+def emptyset():
+    return c2py(<RCP[const symengine.Basic]>(symengine.emptyset()))
+
+
+def universalset():
+    return c2py(<RCP[const symengine.Basic]>(symengine.universalset()))
+
+
+def finiteset(*args):
+    cdef symengine.set_basic s
+    cdef Basic e_
+    for e in args:
+        e_ = sympify(e)
+        s.insert(<RCP[symengine.const_Basic]>(e_.thisptr))
```

Comment:
The failure now is not because of that. Open `symengine_wrapper.cpp` and see which line from `symengine_wrapper.pyx` is causing the problem. You can find the line by scrolling above and looking for the first comment.